var/spack/repos/builtin/packages/py-pygit2/package.py

```diff
@@ -14,15 +14,27 @@ class PyPygit2(PythonPackage):
     homepage = "http://www.pygit2.org/"
     pypi = "pygit2/pygit2-0.24.1.tar.gz"
 
+    version('1.4.0', sha256='cbeb38ab1df9b5d8896548a11e63aae8a064763ab5f1eabe4475e6b8a78ee1c8')
+    version('1.3.0', sha256='0be93f6a8d7cbf0cc79ae2f0afb1993fc055fc0018c27e2bd01ba143e51d4452')
+    version('0.28.2', sha256='4d8c3fbbf2e5793a9984681a94e6ac2f1bc91a92cbac762dbdfbea296b917f86')
     version('0.24.1', sha256='4d1d0196b38d6012faf0a7c45e235c208315672b6035da504566c605ba494064')
 
     extends('python')
     depends_on('py-setuptools', type='build')
     # Version must match with libgit2
     # See: http://www.pygit2.org/install.html
-    depends_on('libgit2@0.24:', when='@0.24:')
+    depends_on('libgit2@1.1:1.1.99', when='@1.4:')
+    depends_on('libgit2@1.0:1.0.99', when='@1.2:1.3.99')
+    depends_on('libgit2@0.99:1.0.99', when='@1.1:1.1.99')
+    depends_on('libgit2@0.28:0.28.99', when='@0.28:1.0.99')
+    depends_on('libgit2@0.24:0.27.99', when='@0.24:0.27.99')
+    depends_on('python@3.6:', when='@1.4.0:')
+    depends_on('python@3.6:3.8.99', when='@1.2:1.3.99')
+    depends_on('python@3.5:3.8.99', when='@1.0:1.1.99')
+    depends_on('python@2.7:3.87.99', when='@0.28:0.99.99')
```

Comment:
```suggestion
    depends_on('python@2.7:3.8.99', when='@0.28:0.99.99')
```

Reply:
'python@2.7:3.7.99' is right for when='@0.28:0.99.99'

Reply:
See b7d0022